terraform/gitlab.tf

```diff
@@ -632,3 +632,94 @@ resource "aws_s3_bucket" "gitlab_bucket" {
     }
   }
 }
+
+locals {
+  ownertag = "kubernetes.io/cluster/${var.cluster_name}"
+}
+
+resource "aws_dlm_lifecycle_policy" "pvbackups" {
+  description        = "${var.cluster_name} pv backups policy"
+  execution_role_arn = aws_iam_role.pvbackups.arn
+  state              = "ENABLED"
+
+  policy_details {
+    resource_types = ["VOLUME"]
+
+    schedule {
+      name = "2 years of daily snapshots"
+
+      create_rule {
+        interval      = 24
+        interval_unit = "HOURS"
+        times         = ["23:45"]
+      }
+
+      retain_rule {
+        count = 104
```

Comment:
Is this 104 weeks or days?

Reply:
That's how many snapshots to keep.  But you are right.  I didn't do the math right!  Thanks!

Reply:
Take a look at it now.  730 days should = 2 years of daily backups.